app/controllers/dashboard_controller.rb

```diff
@@ -1,9 +1,13 @@
 # frozen_string_literal: true
 
 require "velum/kubernetes"
+require "velum/suse_connect"
 
 # DashboardController shows the main page.
 class DashboardController < ApplicationController
+  skip_before_action :authenticate_user!, only: :autoyast
```

Comment:
We cannot serve the autoyast profile to other workers then, they would get redirected to the login page :)